src/debuggers/natvis/magnum.natvis

```diff
@@ -101,4 +161,24 @@
       </ArrayItems>
     </Expand>
   </Type>
+  <!-- Magnum::Math::Deg< Float > -->
+  <Type Name="Magnum::Math::Deg&lt;float&gt;">
+    <DisplayString IncludeView="simple">{_value,g}° ({_value * 3.14159274f / 180.0f,g} radians)</DisplayString>
+    <DisplayString ExcludeView="simple">{_value,g}° ({_value * 3.14159274f / 180.0f,g} radians)</DisplayString>
+  </Type>
+  <!-- Magnum::Math::Deg< Double > -->
+  <Type Name="Magnum::Math::Deg&lt;double&gt;">
+    <DisplayString IncludeView="simple">{_value,h}° ({_value * 3.1415926535897931 / 180.0,h} radians)</DisplayString>
+    <DisplayString ExcludeView="simple">{_value,h}° ({_value * 3.1415926535897931 / 180.0,h} radians)</DisplayString>
+  </Type>
+  <!-- Magnum::Math::Rad< Float > -->
+  <Type Name="Magnum::Math::Rad&lt;float&gt;">
+    <DisplayString IncludeView="simple">{_value,g} radians ({180.0f * _value / 3.14159274f,g}°)</DisplayString>
+    <DisplayString ExcludeView="simple">{_value,g} radians ({180.0f * _value / 3.14159274f,g}°)</DisplayString>
+  </Type>
+  <!-- Magnum::Math::Rad< Double > -->
+  <Type Name="Magnum::Math::Rad&lt;double&gt;">
+    <DisplayString IncludeView="simple">{_value,h} radians ({180.0 * _value / 3.1415926535897931,h}°)</DisplayString>
+    <DisplayString ExcludeView="simple">{_value,h} radians ({180.0 * _value / 3.1415926535897931,h}°)</DisplayString>
+  </Type>
```

Comment:
Is the float/double specialization here really needed? Using the float value of pi even for doubles should be enough for quick and reasonably accurate debug output ☺️

Reply:
Good point. I think using the double value of pi even for floats (i.e., the inverse of your suggestion) wouldn't hurt.

Reply:
I was fine with it but wasn't sure so went the safe route.
Then I'm removing the specializations and using the double pi value as @mosra said 👍